### ISSUE TITLE
Customized Fade In Duration

### ISSUE THREAD

**Issue by Dannydads** [Superkat32/YouWereSlain#5]
Is there any chance you can add a config option to set the fade in delay for the text? I really like the effect, but feel it's too slow with how quickly I like my re spawn to be. Really great mod!

**Comment by Superkat32** [comment]
Yeah, this is something I'd like to add eventually. My goal with the next update is to revamp the config to be easier to navigate quickly, add more QoL features(such as fade in duration), and add some additional compatibility with other mods that change the death screen(if you have any feedback on the config I'd like to hear it). This feature wouldn't really be hard to add, it would just be changing [this line of code](https://github.com/Superkat32/YouWereSlain/blob/739c9232c0cce10bba8deef324882b8e0434f089/src/main/java/net/superkat/youwereslain/mixin/DeathScreenMixin.java#L197C18-L197C18) to use a number from the config. I don't actually know what will happen if you change the duration to too low of a number, but I'd imagine it would probably be fine... _probably._

On that note, however, I wouldn't be surprised if it would take at least a month or two before I'm able to get started on the next update for this mod. So, that leaves us with a few options. You could wait if you wanted, but as mentioned earlier, I have no idea when I'll be able to simply get started on the next update. So, here are our other options.

If you are familiar with Minecraft modding, you can either fork or clone this repo, and change [this line of code](https://github.com/Superkat32/YouWereSlain/blob/739c9232c0cce10bba8deef324882b8e0434f089/src/main/java/net/superkat/youwereslain/mixin/DeathScreenMixin.java#L197C18-L197C18) to the amount of Minecraft ticks you want the fade in to be. 

If you aren't necessarily familiar with modding, but have an understanding of coding in general, I can point you to some resources to help build the mod so you can get the jar file(after forking/cloning and changing the line of code). 

If you aren't familiar with this at all(or don't have the time), I'm happy to do it for you and give you the jar file somewhere here on GitHub. I could commit and push the duration change, wait for the [GitHub actions to build](https://github.com/Superkat32/YouWereSlain/actions), then revert the change. This would give you a zip folder which would contain the jar file you are looking for, all from GitHub. Alternatively(only if you are comfortable with it), I think I could just send the jar file here in this issue discussion. I'm unsure if I can straight up send a jar file, but I know I can send a zip file, which can contain the jar.

If you have any questions, feel free to ask!

**Comment by Dannydads** [comment]
Yea, honestly it's not that big of a deal, and I'm happy to wait however long it is before you can get around to updating. It's been really sweet as is - thank you for the hard work!

**Comment by Superkat32** [comment]
Thanks for the kind words! If you have any other feedback/suggestions(especially on the config screen, because I feel like that needs some work), feel free to mention it here!
